bindings/gumjs/gumdukinterceptor.c

```diff
@@ -714,6 +777,35 @@ GUMJS_DEFINE_GETTER (gumjs_invocation_context_get_depth)
   return 1;
 }
 
+GUMJS_DEFINE_SETTER (gumjs_cached_invocation_context_set_property)
+{
+  GumDukHeapPtr target = _gumjs_duk_require_heapptr (ctx, 0);
+  const gchar * property = duk_safe_to_string (ctx, 1);
+  GumDukHeapPtr receiver = _gumjs_duk_require_heapptr (ctx, 3);
```

Comment:
Let's move these statements after the declarations (we try to stay -Wdeclaration-after-statement -clean).